### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by ulisesedu7**
In this project, I've implemented API calls to send and receive the information from the game scores. I've added modules for each new function and added the events to the index JS file.

### CODE REVIEW

**Comment by ulisesedu7** [comment]
Hi Next Reviewer,

I'm using the word **_dev_** instead of **_start_** to use my webpack server, so if you run `npm run dev` it works. Previous reviewers in the webpack project told me everything was okay with that.

File: src/modules/add-scores.js [hunk 1]
```diff
@@ -4,17 +4,13 @@ const scoreName = document.getElementById('score-input');
 const scoresTable = document.getElementById('scores-table');
 
 class StoredScores {
-  /*
-  Here I'd insert a function to display the stored books if there were any stored somewhere
-  */
-
-  static addScores(scores) {
+  static addScores(firstName, scoreNumber) {
```
Comment (ShahierNashaat) [inline src/modules/add-scores.js:7]: - [x] Please use arrow functions to be like the standard of the ES6.

File: src/modules/add-scores.js [hunk 2]
```diff
@@ -23,6 +19,10 @@ class StoredScores {
     inputName.value = '';
     scoreName.value = '';
   }
+
+  static clearTable() {
+    scoresTable.innerHTML = '';
+  }
```
Comment (ShahierNashaat) [inline src/modules/add-scores.js:25]: - [ ] Please use arrow functions to be like the standard of the ES6.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @ulisesedu7 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/add-scores.js (hunk 2 above)
Comment (ulisesedu7) [reply src/modules/add-scores.js:25]: @ShahierNashaat Hi, thanks for the feedback! I have a question tho, is it mandatory to use an arrow function? I thought classes were already ES6, and those functions you're pointing out right now were already in my previous pull request, that got approved with no mention of that, could you clarify with me? 

I found more useful classes for that specific part but would like to confirm with you, as previous reviewers have accepted my project that way. 

Thanks in advance!

Comment (ShahierNashaat) [reply src/modules/add-scores.js:25]: Yes it is mandatory but if you would like to leave it as it is, it is ok.

Comment (ulisesedu7) [reply src/modules/add-scores.js:25]: @ShahierNashaat I honestly prefer to use it like that, I've gotten pretty familiar with the class's syntax and I like it a lot. In that case, would the project be approved? 

Thanks for the response!

Comment (ShahierNashaat) [reply src/modules/add-scores.js:25]: Actually I don't know if it will be approved or not.

**Review by mikethreels — APPROVED** [review]
Hi Ulises, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**